src/App.tsx

```diff
@@ -44,6 +45,11 @@ function App() {
               <Library library={library} />
             </div>
           </Route>
+          <Route exact path="/free-games">
```

Comment:
Why not Route with component and also why not move the class "content" into the FreeProduct function ?

```suggestion
          <Route exact path="/free-games" component={FreeProducts} />
```